Script/Packages/Atomic/Core.json

```diff
@@ -10,6 +10,11 @@
 			"SendEvent" : ["StringHash"]
 		}
 	},
+	"overloads" : {
+		"Object" : {
+			"UnsubscribeFromEvent" : ["StringHash"]
```

Comment:
We're going top need both flavors of this, unsubscribing from global events and from object specific events (like the SubscribeToEvent works)